src/test/java/io/github/murdos/easyrandom/protobuf/Protobuf3MessageGenerationTest.java

```diff
@@ -77,42 +76,79 @@ void shouldGenerateTheSameValueForTheSameSeed() {
+        assertThat(protoInstance.getSint32Field()).isEqualTo(-1621910390);
+        assertThat(protoInstance.getSint64Field()).isEqualTo(-2298228485105199876L);
+        assertThat(protoInstance.getFixed32Field()).isEqualTo(-1219562352);
+        assertThat(protoInstance.getFixed64Field()).isEqualTo(2992351518418085755L);
+        assertThat(protoInstance.getSfixed32Field()).isEqualTo(-1366603797);
+        assertThat(protoInstance.getSfixed64Field()).isEqualTo(-3758321679654915806L);
         assertThat(protoInstance.getBoolField()).isTrue();
-        assertThat(protoInstance.getStringField()).isEqualTo("eOMtThyhVNLWUZNRcBaQKxI");
+        assertThat(protoInstance.getStringField()).isEqualTo("wSxRIexQAaxVLAiN");
         assertThat(protoInstance.getBytesField().toByteArray()).containsExactly(
-                -35, -15, 33, -71, -107, 4, -68, 60, -47, -116, -85, -3, -86, -16, 51, 77,
-                22, -47, -41, 64, 50, 38, -6, -110, 69, 87, -38, -101, 58, 15, 70, 66
+                53,114,79,60,-14,-35,50,97,116,107,41,53,-39,-28,114,79,-111,
+                98,-14,-11,-97,102,-22,83,-126,104,-108,-59,-97,93,-122,-67
         );
+
         assertThat(protoInstance.getEnumField()).isEqualTo(Proto3Enum.SECOND_VALUE);
         assertThat(protoInstance.getStringValueField())
                 .isNotNull()
-                .extracting(StringValue::getValue).isEqualTo("eOMtThyhVNLWUZNRcBaQKxI");
+                .extracting(StringValue::getValue).isEqualTo("tg");
         assertThat(protoInstance.getRepeatedStringFieldList()).containsExactly(
-                "yedUsFwdkelQbxeTeQOvaScfqIOOmaa",
-                "JxkyvRnL",
-                "RYtGKbgicZaHCBRQDSx",
-                "VLhpfQGTMDYpsBZxvfBoeygjb",
-                "UMaAIKKIkknjWEXJUfPxxQHeWKEJ"
+                "AJVH",
+                "WuGaTPB",
+                "NuGSIFWDPVPqKClkqNpxLIRO",
+                "jukCwoSTgRGMwWnAeflhVmclqMX",
+                "bWyqZZW"
         );
 
         assertThat(protoInstance.hasEmbeddedMessage()).isTrue();
         assertThat(protoInstance.getEmbeddedMessage()).satisfies(embeddedMessage -> {
-            assertThat(embeddedMessage.getStringField()).isEqualTo("eOMtThyhVNLWUZNRcBaQKxI");
-            assertThat(embeddedMessage.getEnumField()).isEqualTo(Proto3Enum.SECOND_VALUE);
+            assertThat(embeddedMessage.getStringField()).isEqualTo("LRHCsQ");
+            assertThat(embeddedMessage.getEnumField()).isEqualTo(Proto3Enum.UNKNOWN);
+        });
+        assertThat(protoInstance.getOneofFieldCase()).isEqualTo(Proto3Message.OneofFieldCase.THIRDCHOICE);
+    }
+
+    @Test
+    void shouldGenerateDifferentObject() {
+        EasyRandomParameters parameters = new EasyRandomParameters()
+                .seed(123L)
+                .collectionSizeRange(3, 10);
+        EasyRandom easyRandom = new EasyRandom(parameters);
+
+        Proto3Message protoInstance = easyRandom.nextObject(Proto3Message.class);
```

Comment:
Could you rename it to `firstInstance`, please?
```suggestion
        Proto3Message firstInstance = easyRandom.nextObject(Proto3Message.class);
```